### ISSUE TITLE
Track translations

### ISSUE THREAD

**Issue by iandunn** [WordPress/five-for-the-future#196]
This one might impact the most contributors

we'll probably want to do some kind of digest though, to avoid folks having dozens of profile entries on a single day, which would crowd out all the other stuff on their profile.  the details for that are discussed in #197 

need to get input from the polyglots team to make sure this lines up w/ what they want

- [x] Suggesting a translation
- [x] Getting a suggested translation approved
- [x] Handle bulk translations more efficiently
- [x] Reviewing (= both rejecting and approving) translation
- [x] Switch to `dashicons-translation` icon
- [ ] Becoming a PTE for a project
- [ ] Assigning a PTE/GTE/CLPTE/Locale Manager role to someone
- [ ] Handle imports (blocked by https://github.com/GlotPress/GlotPress/issues/1467)

**Comment by iandunn** [comment]
Do you all have any thoughts on what contributions the Polyglots team wants to track on w.org/profiles? @vlad-timotei @amieiro @naokomc @angelasjin

**Comment by naokomc** [comment]
I think we should count:

**Everyone**
- Suggesting a translation (especially for less experienced contributors, to encourage the action)
- Getting a suggested translation approved (this is more ideal metric to aim long term)
- Becoming a PTE for a project

_We'll need to avoid providing incentives to just add many low-quality (not approvable) translations, but at the same time need to take into account sometimes approval doesn't happen because GTEs are inactive or very picky._

**Global Mentor / Locale Managers / G&PTEs**
- Reviewing (= both rejecting and approving) translation
- Assigning a PTE/GTE/CLPTE/Locale Manager role to someone

And for a digest, how about one of these?

- Use a milestone, like the first 1 translation approved, then 100, 1,000, 10,000...
- Tally up the count at the end of each day and post once
- For GTEs and PTEs, it might make sense to combine reviewed strings and translated strings

Not sure if you are looking for these, but here are other items for consideration for polyglots contributors.

- Publishing a blog post on the Rosetta site
- Publishing or editing a page on the Rosetta site
- Publishing or editing a Rosetta /team/ post or handbook page
- Publishing or editing a localized HelpHub page (e.g. https://xx.wordpress.org/support/article/xxx)
- Replying to a local forum topic
- Moderating a local forum topic or reply
- Moderating or commenting on a post on Polylgots P2
- Publishing or editing a Polyglots handbook page

**Comment by iandunn** [comment]
Those seem great, thanks!

> Assigning a PTE/GTE/CLPTE/Locale Manager role to someone

Should that include when someone's role is removed or downgraded? [Other teams are discussing that as well](https://github.com/WordPress/five-for-the-future/issues/193#issuecomment-1164806309).

> And for a digest, how about one of these?

That could work, I'll bring it up in #197 so we can keep that topic centralized.

> other items for consideration

I think these are already working, or are already planned in other tickets:
* Publishing a blog post on the Rosetta site
* Publishing or editing a page on the Rosetta site
* Publishing or editing a Rosetta /team/ post or handbook page 
* Publishing or editing a localized HelpHub page (e.g. https://xx.wordpress.org/support/article/xxx)
* Replying to a local forum topic
*  commenting on a post on Polylgots P2
* Publishing or editing a Polyglots handbook page

I don't think these are yet (for Rosetta or english sites)

* Moderating a post on Polylgots P2
* Moderating a local forum topic or reply

**Comment by naokomc** [comment]
Similar to Angela's suggestion about WordCamp/Meetup status change, I think removal/downgrade action by Locale Managers (or Global Mentors) should also be counted too.

**Comment by iandunn** [comment]
Do these sound like the strings you want to use in those situations?

Manager's profile: 
> Removed @iandunn as a Project Translation Editor on translate.wordpress.org

Profile of the person being removed:
> Was removed as a Project Translation Editor on translate.wordpress.org

Or were you thinking the first message would leave the person's name out, and that the second one wouldn't be posted?

If we do post them, I think it'll be important to pick the language carefully, since it could be awkward/uncomfortable for some folks to have their profile broadcast that they were removed from a position.

**Comment by tellyworth** [comment]
It seems potentially problematic to me to publicly record removal/downgrade actions, with or without names.

> Suggesting a translation (especially for less experienced contributors, to encourage the action)
> Getting a suggested translation approved (this is more ideal metric to aim long term)

Perhaps it would be enough to just record the first occurrence of these for each user?

**Comment by iandunn** [comment]
> It seems potentially problematic to me to publicly record removal/downgrade actions, with or without names.

Yeah, it seems a bit impolite to me, although I realize that might be culturally dependent.

> Perhaps it would be enough to just record the first occurrence of these for each user?

🤔 It'd be more work to make an exception from #197, and could be confusing to folks ("I suggested 20 strings, why does it only say 1?"). I'm not sure there's a big benefit to it, since it'll be obvious from a person's profile whether or not their suggestions are being approved.

**Comment by NekoJonez** [comment]
I'm adding my two cents for a future version of this and I'd love to see a separation per locale. Since some polyglots translate for more then one locale.

**Comment by NekoJonez** [comment]
Reviewing, maybe check with @ocean90 and @amieiro on that since the feedback module might be interesting to hook into.

----

Besides that, what about fuzzy strings? Eg, if a pte/gte sets strings on fuzzy so he/she/they can talk to the translator for improvements?

**Comment by iandunn** [comment]
Marking a string as fuzzy is counted as a "review" action. Were you thinking about that, or something else?

**Comment by NekoJonez** [comment]
> Marking a string as fuzzy is counted as a "review" action. Were you thinking about that, or something else?

Nope, exactly that. Wanted to mention it since I saw the checklist and didnt see fuzzy in there so I wanted to double check

**Comment by iandunn** [comment]
👍🏻

**Comment by NekoJonez** [comment]
Somewhat related: https://meta.trac.wordpress.org/ticket/6414

**Comment by pedro-mendonca** [comment]
Hi, not sure if it's the right place.

I would like to suggest the use of the Polyglots dashicon for the translation activity entries.

Currently use `dashicons-admin-generic` - `f111`:
![imagem](https://user-images.githubusercontent.com/7371591/181254674-25fae394-ffe8-451d-b6cb-95d04877cd88.png)

I suggest the `dashicons-translation` - `f326`:
![imagem](https://user-images.githubusercontent.com/7371591/181255656-7c86d6fa-d7d4-45ac-a456-77de3b3fafcf.png)

**Comment by iandunn** [comment]
That's a good idea 👍🏻

**Comment by dd32** [comment]
> I would like to suggest the use of the Polyglots dashicon for the translation activity entries.

@pedro-mendonca Done!

**Comment by jdxo** [comment]
Hi folks!

I recently opened a ticket on the Meta Trac regarding imported strings with the "Current" status not being logged as contributions on user profiles - I was directed here.

As a Locale Manager I tend to import strings as "Current" fairly often, so I would love to see that implemented/fixed if at all possible!

**Comment by tobifjellner** [comment]
@jdxo 
You also might mention this to @yaniiliev and perhaps bring it up in Slack channel meta-janitors ( https://wordpress.slack.com/archives/C0ATCQF2PMM ) where currently a lot of things around fixing profile pages, etc. are happening.

**Comment by tobifjellner** [comment]
Uups. Thought we were on Meta Trac. :) @yani-  is a better match :)

**Comment by wppoland** [comment]
The "Handle imports" item above is no longer blocked. GlotPress#1467 was closed as completed on 2026-07-28 by GlotPress#2033, which added `$created_translation_ids` to `gp_translations_imported` for exactly this purpose.

One gap that PR leaves, worth knowing before wiring imports up here: when an import matches a translation already waiting from someone else, GlotPress approves that row rather than creating one, so the row never appears in `$created_translation_ids` and the original translator drops out of the credit set. GlotPress#2048 adds a third `$approved_translation_ids` argument for that case. If import activity is built on the created list alone, it will miss those.

Context on the profile-side symptom: https://meta.trac.wordpress.org/ticket/8372.
